src/main/java/de/tum/in/www1/artemis/web/rest/dto/competency/LearningPathNavigationOverviewDto.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.in.www1.artemis.web.rest.dto.competency;
+
+import java.util.List;
+
+public record LearningPathNavigationOverviewDto(List<LearningPathNavigationObjectDTO> learningObjects) {
```

Comment:
I think we have the convention to use `DTO` and not `Dto`
There should be an architecture test to verify that, at least in my recent PR where I made this mistake, a test failed

Reply:
I wonder if this record is actually needed. Do you really need a wrapper for a list? or you plan to extend this in the future? (then a comment might be helpful)